src/stubs/angular/make-angular-component.mts

```diff
@@ -0,0 +1,25 @@
+import * as fs from 'fs';
+import { writeFile } from '../../utils/utils.mjs';
+
+export function makeAngularComponent(filePathDestination: string, templateFilePath: string, componentName: string): void {
+	fs.readFile(templateFilePath, 'utf8', (err, component) => {
```

Comment:
since makeAngularComponent is used already inside the read function, I don't think you need to read that again, you could simply pass the content of the file, you're going to to update and write 😄